vllm/platforms/cpu.py

```diff
@@ -86,4 +86,10 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
                            parallel_config.distributed_executor_backend)
             parallel_config.distributed_executor_backend = "mp"
         if parallel_config.worker_cls == "auto":
-            parallel_config.worker_cls = "vllm.worker.cpu_worker.CPUWorker"
+            if vllm_config.speculative_config:
+                parallel_config.worker_cls = \
+                    "vllm.spec_decode.spec_decode_worker.create_spec_worker"
+                parallel_config.actual_worker_cls = \
```

Comment:
can we rename the field to `sd_worker_cls` ?

Reply:
maybe add a field `sd_worker_cls` to `ParallelConfig` , and default to `None`.

Reply:
Will do